code-challenges.js

```diff
@@ -13,52 +13,139 @@ var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
 
+//Creating a function that will take in a number and attempt to divide it by three and return a pleasant little string along with it. 
+const divisibleBy = (num) =>{
+    if(num / 3){
+        return `${num} is divisible by three!`
+    } else { return `${num} is not divisible by three  :( `
+    }
+}
+
+//Logging the function call three times, with the set variables above as arguments. I know there is a better way of doing this, but just wanted to do the long way
```

Comment:
this is a totally reasonable way to check all these variables, no problem in my opinion :)